config/settings/base.py

```diff
@@ -76,6 +76,7 @@
 LOCAL_APPS = [
     "ats.users.apps.UsersConfig",
     "ats.companies",
+    "ats.jobs.apps.JobsConfig",
```

Comment:
```suggestion
    "ats.jobs",
```
To be consistent, what you did is not wrong